src/services/mongoose/schema.ts

```diff
@@ -192,6 +192,16 @@ const AccountSchema = new Schema<AccountRecord>(
         partialFilterExpression: { username: { $type: "string" } },
       },
     },
+    npub: {
+      type: String,
+      minlength: 63,
+      maxlength: 63,
+      index: {
+        unique: true,
+        collation: { locale: "en", strength: 2 },
+        partialFilterExpression: { npub: { $type: "string" } },
```

Comment:
I'm no expert in databases and indexing, but I'm not sure how well this index will perform since all fields begin with "npub1" (low cardinality). 

Either way, not really a high concern for now as we don't have many users, but maybe something to keep an eye on.